packages/cli/test/services/environment.test.ts

```diff
@@ -38,12 +37,15 @@ describe('environment service', (): void => {
 
     describe('process.env.BOOSTER_ENV set', (): void => {
       beforeEach(() => {
-        process.env.BOOSTER_ENV = 'testing'
-        stub(process.env, 'BOOSTER_ENV').value('testing')
+        process.env.BOOSTER_ENV = ''
+        replace(process.env, 'BOOSTER_ENV', 'testing')
+      })
+
+      afterEach(() => {
+        restore()
```

Comment:
This is not actually needed because we're handling the environment variable state manually at the beginning of this file

```suggestion
        process.env.BOOSTER_ENV = 'testing'
```